pkg/machine/qemu/config.go

```diff
@@ -21,6 +21,16 @@ type QEMUVirtualization struct {
 	machine.Virtualization
 }
 
+func setVMFile(dest *machine.VMFile, path string) error {
```

Comment:
I don't see a reason to have this function?

Reply:
I noticed we did the same op to set different vm files, so I _tried_ to make it reusable. I can just get rid of it and close my applehv PR if you don't think its necessary

Reply:
Fixed

Reply:
As mentioned in the other PR, I _really_ don't like IN/OUT arguments.  Either, we use a receiver (i.e., add a `Set(..)` method to `VMFile`) OR we turn it into `setVMFile(path string) (dest *machine.VMFile, error)`.